pkgs/development/compilers/spirv-llvm-translator/force-llvm-19.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 6252c5da3..a90a7017a 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,7 +1,7 @@
+ cmake_minimum_required(VERSION 3.13.4)
+ 
+ if(NOT DEFINED BASE_LLVM_VERSION)
+-  set (BASE_LLVM_VERSION 19.0.0)
++  set (BASE_LLVM_VERSION 19.1.0)
```

Comment:
Can we not set this without a patch, since it checks whether it's already defined?

Reply:
Done